### ISSUE TITLE
Research and fix any Cumulative Layout Shift issues

### ISSUE THREAD

**Issue by RichardTaylor** [mysociety/alaveteli#7178]
The [Google Search Console](https://search.google.com/search-console) for WhatDoTheyKnow is showing a warning/suggestion ("Status: Need improvement") in respect of "Cumulative Layout Shift (CLS)".

The linked advice explains what that relates to:

> Have you ever been reading an article online when something suddenly changes on the page? Without warning, the text moves, and you've lost your place.

https://web.dev/cls/

The issue appears to be on request pages on desktop, but there are scant details. 

It might be worth trying to understand this from the point of view of seeking better search optimisation, even if we don't have an issue which actually impacts users. 

Related:

* https://github.com/mysociety/whatdotheyknow-theme/issues/1296

**Comment by RichardTaylor** [comment]
I tried a [Cumulative Layout Shift Debugger (CLS)](https://webvitals.dev/cls).

https://webvitals.dev/cls/UMevX7565usOlUjYUCGP

The results leave me wondering if there's an issue with the automated detection of issues. This might be a false positive rather than a real issue impacting human users. 

If Google cares though it might be worth seeking to resolve the claimed issues.

**Comment by FOIMonkey** [comment]
I've observed elements moving about when request pages are slow to load for whatever reason. The menu on the actions button at the top appears fully extended and the side bar with the related requests is not present, making the actual request part of the page jump about. The issue is intermittent so hard to document.

**Comment by WilliamWDTK** [comment]
Here's a video (I artificially restricted the bandwidth to `google-chrome` to 10 KB/s):

[Slow loading page](https://user-images.githubusercontent.com/99501342/180493833-4ebf2d19-efea-4151-bf31-8e7dd48da0a2.webm)

**Comment by HelenWDTK** [comment]
This issue has been automatically closed due to a lack of discussion or resolution for over 12 months.
Should we decide to revisit this issue in the future, it can be reopened.
